less/icons/icons-png-fallback.css

```diff
@@ -222,6 +222,8 @@
 
 .fuelux-icon-multimail, .glyphicon-multimail { background-image: url('png/multimail.png'); background-repeat: no-repeat; .fuelux-icon; } 
 
+.fuelux-icon-new-window, .glyphicon-new-window { background-image: url('png/new-window.png'); background-repeat: no-repeat; .fuelux-icon; } 
```

Comment:
I don't see `png/new-window.png` in this PR anywhere... is it supposed to be here?